kevm-pyk/src/kevm_pyk/utils.py

```diff
@@ -41,10 +41,11 @@ def parallel_kcfg_explore(
     kprove: KProve,
     proof_problems: Dict[str, KCFG],
     save_directory: Optional[Path] = None,
-    max_depth: int = 100,
+    max_depth: int = 1000,
```

Comment:
Consider

```suggestion
    max_depth: Optional[int] = None,
```

and setting the default `int` value in the function body. Then callers don't need to be aware of the actual default value.

Reply:
I feel like the default value gives people an idea of what a "sane" value here is though. Otherwise someone might think "oh, maybe 10 is a good value", or they may think "maybe 1000000 is a good value"?